### PR TITLE
Make sure every PR syncs generated files and generate schema again

### PR DESCRIPTION
**Pull request by mihaimaruseac**
We bumped the gqlgen dependency in https://github.com/guacsec/guac/pull/368, so the generated files changed. We are bringing them back to date now. Also making sure that future CI will test that generated files are up to date.

Cleaned up Makefile formatting while here, making it more consistent.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,34 +8,46 @@ LDFLAGS="-X $(PKG).version=$(VERSION) -X $(PKG).commit=$(COMMIT) -X $(PKG).date=
 .DEFAULT_GOAL := build
 
 .PHONY: all
-all: test cover fmt lint ci build generate
+all: test cover fmt lint ci build generate generated_up_to_date
```
Comment (lumjjb) [inline Makefile:11]: hmm i think this wouldn't need it, since it is part of the `ci`?

**Review by lumjjb — APPROVED** [review]
minor nit, else lgtm

File: Makefile (hunk 1 above)
Comment (mihaimaruseac) [reply Makefile:11]: Right, let me remove.

**Review by pxp928 — APPROVED** [review]
LGTM

**Pull request merged**